src/numerics.jl

```diff
@@ -90,37 +107,79 @@ function N(b::BasicType{Val{:Integer}})
     end
 end
 
-N(b::Basic) = N(BasicType(b))
+N(b::BasicType{Val{:Rational}}) = Rational(N(num(b)), N(den(b))) # TODO: Implement rational_get_mpq in cwrapper.h
```

Comment:
This is now implemented, we will have to do this conditionally though as symengine-v0.2 does not have it.